packages/craco/lib/plugins.js

```diff
@@ -0,0 +1,31 @@
+function pluginByName(targetPluginName) {
+    return plugin => {
+        return plugin.constructor.name === targetPluginName;
+    };
+}
+
+function getPlugin(webpackConfig, matcher) {
+    const matchingPlugin = webpackConfig.plugins.find(matcher);
+
+    return {
+        isFound: matchingPlugin !== undefined,
+        match: matchingPlugin
+    };
+}
+
+function removePlugins(webpackConfig, matcher) {
+    const prevCount = webpackConfig.plugins.length;
+    webpackConfig.plugins = webpackConfig.plugins.filter(x => !matcher(x));
+    const removedPluginsCount = prevCount - webpackConfig.plugins.length;
+
+    return {
+        hasRemovedAny: removedPluginsCount > 0,
+        removedCount: removedPluginsCount
+    };
+}
+
+module.exports = {
+    getPlugin,
+    pluginByName,
+    removePlugins
+};
```

Comment:
That's very cool. We could make this available to plugins author.

Reply:
Maybe rename to "webpack-plugins" ? it's confusing since plugins is could also be for "craco plugins"

Reply:
i renamed the file, but kept the method name as is